src/app/models/visit.ts

```diff
@@ -1,5 +1,5 @@
 export interface Visit {
     patientId: string;
     note: string;
-    points: number[];
+    points: string[];
```

Comment:
why string array, may be because the  _id field in backend is string and objects in all backend is array

Reply:
this will change later on as we going to be storing a different things for visit